non-critical-infra/hosts/umbriel/README.md

```diff
@@ -5,7 +5,7 @@
 If you recreate `umbriel`, it will generate a new `DKIM` signature. That's ok to
 do, but you'll need to update the corresponding `mail._domainkey.*` `TXT` DNS
 record in `terraform/dns.tf` with the generated key in
-`/var/dkim/mail-test.nixos.org.mail.txt`.
+`/var/dkim/nixos.org.mail.txt`.
```

Comment:
Maybe we can install this secret file with sops-nix? Than it's less stateful to generate the secret if we ever move the machine.

Reply:
This is what the module does:

```
              ${pkgs.opendkim}/bin/opendkim-genkey -s "${cfg.dkimSelector}" \
                                                   -d "${dom}" \
                                                   --bits="${toString cfg.dkimKeyBits}" \
                                                   --directory="${cfg.dkimKeyDirectory}"
              mv "${cfg.dkimKeyDirectory}/${cfg.dkimSelector}.private" "${dkim_key}"
              mv "${cfg.dkimKeyDirectory}/${cfg.dkimSelector}.txt" "${dkim_txt}"
```

Reply:
I'd love to do this. Two ideas:

- get https://gitlab.com/simple-nixos-mailserver/nixos-mailserver/-/merge_requests/344 merged, or use a branch of SNM
- (I *think* this would work) add a system activation script to create this file

Which approach would you like me to explore?

Reply:
We can keep this as it is for now.